flagsmith/validate_feature_state_value.go

```diff
@@ -0,0 +1,35 @@
+package flagsmith
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+func validateFeatureStateValue() validatorFeatureStateValue {
+	return validatorFeatureStateValue{}
+}
+
+type validatorFeatureStateValue struct {
+}
+
+func (v validatorFeatureStateValue) Description(ctx context.Context) string {
+	return "One of string_value, integer_value or boolean_value must be set"
+}
+
+func (v validatorFeatureStateValue) MarkdownDescription(ctx context.Context) string {
+	return "One of string_value, integer_value or boolean_value must be set"
+}
+
+func (v validatorFeatureStateValue) ValidateObject(ctx context.Context, req validator.ObjectRequest, resp *validator.ObjectResponse) {
+
+	attrs := req.ConfigValue.Attributes()
+
+	if (attrs["string_value"].IsNull() || attrs["string_value"].IsUnknown()) && (attrs["integer_value"].IsNull() || attrs["integer_value"].IsUnknown()) && (attrs["boolean_value"].IsNull() || attrs["boolean_value"].IsUnknown()) {
```

Comment:
More of a general Go question here but is there not a better way to do this? I guess in python it would be something like: 

```python
if not all(attrs.get(required_attr) for required_attr in ("string_value", "integer_value", "boolean_value")):
    # raise validation err
```